trackman/templates/library/index.html

```diff
@@ -18,6 +18,14 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
+<!-- TODO beautify -->
```

Comment:
I'd prefer to do this before merging this, rather than after.

Reply:
agreed. I just wasn't sure where to look in bootstrap and getting it working was priority 0.